client/src/components/History/Content/contentState.js

```diff
@@ -0,0 +1,67 @@
+/*
+    Client representation of state and state messages. See: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/model/__init__.py#L3292
+    for a list of available states.
+*/
+export default {
+    /** deleted while uploading */
+    discarded: {
+        status: "danger",
+        text: "The job creating this dataset was cancelled before completion.",
```

Comment:
I've got downstream tests that I've reworked for the migration to this file from the individual Vue templates. But this state message doesn't seem to get rendered anymore? (I cannot find any reference to these text entries.) Is this intentional - I feel like this was valuable information and would be even more valuable in the case of remote data?

If we don't want this text rendered I can rework the UI, I just want to make sure it is intentional though.

Reply:
Yes, we should display this message if we currently do not. I can do that for you, the logic should be either placed into `ItemContent` or if the message should only be shown upon expansion of datasets into the `DatasetDetails` component.

Reply:
@jmchilton thanks again for pointing that out, I added it to the follow-up PR: #13512, with a little but luck it should be in dev in a couple days.